frappe_private_comment/public/js/controls/replies.js

```diff
@@ -0,0 +1,290 @@
+}
+
+function render_replies(commentSelector, replies) {
+  const $comment = $(commentSelector);
+  const $prevContainer = $comment.next(".threaded-reply-container");
+  $prevContainer && $prevContainer.remove();
+
+  const $replyContainer = $(
+    '<div class="threaded-reply-container " style="margin-left: 90px; position: relative;"></div>'
+  );
+
+  // Add vertical line
+  $replyContainer.append(
+    '<div class="vertical-line" style="position: absolute; left: -30px; top: 0; bottom: 0; width: 1px; background-color: #d1d8dd;"></div>'
+  );
+
+  replies.forEach((reply) => {
+    const $replyContent = $(`
+            <div style="position: relative;">
+                <div class="timeline-badge" style="position: absolute; left: -43px; top: 10px; background-color: #F3F3F3; padding: 5px; border-radius: 999px;">
+                    <svg class="icon icon-md">
+                        <use href="#icon-small-message"></use>
+                    </svg>
+                </div>
+                <div class="timeline-item frappe-card" data-doctype="Comment" id="comment-${reply.name}" data-name="${
+      reply.name
+    }" style="position: relative; background-color: #F7F7F7; max-width: 700px">
+                    <div class="timeline-content">
+                        <div class="timeline-message-box">
+                            <span class="text-muted">
+                                ${frappe.avatar(frappe.session.user, "avatar-medium")}
+                                <span class="timeline-user">${
+                                  reply.comment_by === frappe.session.user ? "You" : reply.comment_by
+                                } commented . </span>
+                                <span>&nbsp; ${frappe.datetime.comment_when(reply.creation)}</span>
+                            </span>
+                            <span>${update_the_comment_visibility(true)}</span><hr />
+                            <div class="read-mode">
+                                <p>${reply.content}</p>
+                            </div>
+                            <div class="edit-mode" style="display: none;">
+                            <textarea class="form-control edit-textarea" rows="3">${reply.content}</textarea>
+                            <div class="mt-2">
+                              <button class="btn btn-sm btn-primary save-edit">Save</button>
+                              <button class="btn btn-sm btn-default cancel-edit">Cancel</button>
+                            </div>
+                          </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        `);
+
+    $replyContainer.append($replyContent);
+
+    const actionButtons = $("<div>").addClass("comment-actions").css({
+      float: "right",
+      marginLeft: "auto",
+    });
+
+    const replyButton = $("<button>")
+      .addClass("btn btn-xs btn-link reply-btn")
+      .css({ "font-size": "12px" })
```

Comment:
Will this add inline style?

Reply:
Update: will separate in a different css file (after existing bugs are fixed)